app/ui-login/client/reset-password/resetPassword.js

```diff
@@ -47,6 +47,24 @@ const resetPassword = (token, password) => new Promise((resolve, reject) => {
 	});
 });
 
+async function setUserPassword(password) {
+	try {
+		const result = await call('setUserPassword', password);
+		if (!result) {
+			return toastr.error(t('Error'));
+		}
+
+		Meteor.users.update({ _id: Meteor.userId() }, {
+			$set: {
+				requirePasswordChange: false,
+			},
+		});
+	} catch (e) {
+		console.error(e);
```

Comment:
Still need this console.log?

Reply:
I left it there on purpose to help debugging in case an unexpected error happen.

I can remove it if it is not helpful =)